src/renderer/features/governance/components/ReferendumDetails/ProposerName.tsx

```diff
@@ -30,21 +31,26 @@ export const ProposerName = ({ referendum }: Props) => {
     fn: (loading, [proposer]) => loading && !proposer,
   });
 
-  if (!isProposerLoading && !proposer) {
-    return null;
-  }
-
-  const proposerName = proposer ? (
+  const proposerName = proposer?.parent ? (
     <AccountAddress
       addressFont="text-text-secondary"
       size={16}
       address={proposer.parent.address}
       name={proposer.parent.name || proposer.email || proposer.twitter || proposer.parent.address}
     />
+  ) : (referendum as OngoingReferendum)?.submissionDeposit?.who ? (
```

Comment:
let's check if it's really ongoing referendum by `referendumService.isOngoing`

Reply:
Much better, thanks.